language-server/modules/langserver-core/spotbugs-exclude.xml

```diff
@@ -82,4 +82,7 @@
     <Match>
         <Class name="org.ballerinalang.langserver.extensions.ballerina.document.BallerinaServiceListResponse" />
     </Match>
+    <Match>
+        <Class name="org.ballerinalang.langserver.symbols.SymbolFindingVisitor" />
```

Comment:
Do we need to skip the whole `SymbolFindingVisitor ` class?. Shall we add the specific bug details such as `Bug pattern` and `Method`.